modules/autoattach.cpp

```diff
@@ -116,22 +118,36 @@ class CChanAttach : public CModule {
         }
     }
 
+    void HandleSwap(const CString& sLine) {
+        u_int iA = sLine.Token(1).ToUInt();
+        u_int iB = sLine.Token(2).ToUInt();
+
+        Swap(iA, iB);
+    }
+
     void HandleList(const CString& sLine) {
+        int Index;
```

Comment:
`iIndex`? please follow the style around...

Reply:
Will push a fix for this and other some of the other issues soon.